tests/requirements.txt

```diff
@@ -0,0 +1,5 @@
+pytest==6.2.4
+jina==2.1.4
```

Comment:
why did you duplicate the main reqs into the test reqs? Just add `pytest` and the requirements that would be needed for tests, apart from the core ones in the main `reqs.txt`